syntax = "proto3";
package DataCenter;

// DataCenter function is start by 201-300
enum Functions{
    Invalid=0;
    reserved 1 to 200;
    RunSQL=201;
    reserved 300 to max;
}


message RunSQLRequest{
    string SQLCMD=1;
}

message RunSQLResponse{
    int32 AffectedRows=1;
    repeated string Results=2;
}